Mlem/Models/Favorite Community.swift

```diff
@@ -14,3 +14,11 @@ struct FavoriteCommunity: Identifiable, Codable, Equatable {
 
     let community: APICommunity
 }
+
+extension FavoriteCommunity: Comparable {
+    static func < (lhs: FavoriteCommunity, rhs: FavoriteCommunity) -> Bool {
+        let lhsFullCommunity = lhs.community.name + (lhs.community.actorId.host ?? "")
+        let rhsFullCommunity = rhs.community.name + (rhs.community.actorId.host ?? "")
+        return lhsFullCommunity < rhsFullCommunity
+    }
```

Comment:
Since `APICommunity` is now `Comparable` as well, and that's what's being compared, would it be more readable to just do `return lhs.community < rhs.community`?